[2023-07-13 18:03:23,395] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.crear_tabla manual__2023-07-13T18:03:08.338527+00:00 [queued]>
[2023-07-13 18:03:23,600] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.crear_tabla manual__2023-07-13T18:03:08.338527+00:00 [queued]>
[2023-07-13 18:03:23,601] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-13 18:03:23,607] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-07-13 18:03:23,611] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-13 18:03:23,688] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): crear_tabla> on 2023-07-13 18:03:08.338527+00:00
[2023-07-13 18:03:23,696] {standard_task_runner.py:52} INFO - Started process 2607 to run task
[2023-07-13 18:03:23,730] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_con_conexion_postgres', 'crear_tabla', 'manual__2023-07-13T18:03:08.338527+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_paises.py', '--cfg-path', '/tmp/tmpfzpsovke', '--error-file', '/tmp/tmppvrqweu9']
[2023-07-13 18:03:23,746] {standard_task_runner.py:80} INFO - Job 39: Subtask crear_tabla
[2023-07-13 18:03:27,156] {task_command.py:371} INFO - Running <TaskInstance: dag_con_conexion_postgres.crear_tabla manual__2023-07-13T18:03:08.338527+00:00 [running]> on host bd4a51b00e09
[2023-07-13 18:03:29,339] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Belenpintor
AIRFLOW_CTX_DAG_ID=dag_con_conexion_postgres
AIRFLOW_CTX_TASK_ID=crear_tabla
AIRFLOW_CTX_EXECUTION_DATE=2023-07-13T18:03:08.338527+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-07-13T18:03:08.338527+00:00
[2023-07-13 18:03:29,359] {logging_mixin.py:115} INFO - creando conexion
[2023-07-13 18:03:38,277] {script_etl.py:20} INFO - DB_USER probando usuario 2: bapintor_coderhouse
[2023-07-13 18:03:38,281] {logging_mixin.py:115} INFO - usuario probando 2 bapintor_coderhouse
[2023-07-13 18:03:39,746] {logging_mixin.py:115} INFO - Conexi√≥n exitosa a la base de datos
[2023-07-13 18:03:40,317] {python.py:173} INFO - Done. Returned value was: None
[2023-07-13 18:03:40,487] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_con_conexion_postgres, task_id=crear_tabla, execution_date=20230713T180308, start_date=20230713T180323, end_date=20230713T180340
[2023-07-13 18:03:40,754] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-13 18:03:41,043] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
